src/hotspot/share/prims/jvmtiThreadState.cpp

```diff
@@ -82,8 +82,10 @@ JvmtiThreadState::JvmtiThreadState(JavaThread* thread, oop thread_oop)
   _earlyret_oop = NULL;
 
   _jvmti_event_queue = NULL;
+#if INCLUDE_JVMTI
   _is_in_VTMT = false;
   _is_virtual = false;
+#endif
```

Comment:
I'm not sure this change is really needed.
My understanding is that the `jvmtiThreadState.cpp` is fully excluded in `non-INCLUDE_JVMTI` mode.